src/java.desktop/windows/native/libawt/java2d/d3d/D3DRenderQueue.cpp

```diff
@@ -866,7 +866,8 @@ void D3DRQ_FlushBuffer(void *pParam)
 
     if (!JNU_IsNull(env, pFlush->runnable)) {
         J2dTraceLn(J2D_TRACE_VERBOSE, "  executing runnable");
-        JNU_CallMethodByName(env, NULL, pFlush->runnable, "run", "()V");
+        jboolean ignoreException;
+        JNU_CallMethodByName(env, &ignoreException, pFlush->runnable, "run", "()V");
```

Comment:
What is the purpose of this change? the only difference is that in the second case the ExceptionCheck will be called, does it affect something?

Reply:
Yes, the `ExceptionCheck()` call will silence the warnings from `-Xcheck:jni`.

Reply:
Does it actually suppress the "Xcheck:jni" or it clears a raised exception? If an exception is still "raised" after this call we should do some additional steps to log/clean it.

Reply:
Yes, the exception will still be "raised" after this call. Since there are no JNI calls (at least those that are forbidden to make with an exception pending) between here and return back to Java, I believe no additional steps are necessary.

Reply:
Then I suggest logging them via some of the trace macros.

Reply:
Sure, added some logging.